docs/resources/resources.md

```diff
@@ -7,7 +7,6 @@ title:  "Resources"
 
 _These articles reflect the state of ZIO at the time of their publication. The code samples might be outdated, considering ZIO was early in development at the time they were written. However, the concepts are still relevant._
 
-- [ZIO with http4s, Auth, Codecs and zio-tests (RC18)](https://timpigden.github.io/_pages/zio-http4s/intro.html) by Tim Pigden (April 2020)
```

Comment:
Would you mind reverting this change? Otherwise looks great! Thanks for adding!

Reply:
Done

Reply:
Thanks!